src/cairoWriter/writers/variableDeclarationStatementWriter.ts

```diff
@@ -68,17 +69,17 @@ export class VariableDeclarationStatementWriter extends CairoASTNodeWriter {
         return [writer.write(getDeclarationForId(id))];
       }
     });
+    const solTyp = safeGetNodeType(node.vInitialValue, this.ast.inference);
+    const cairoTyp = CairoType.fromSol(solTyp, this.ast);
```

Comment:
If `cairoTyp` is only going to be used for it's string representation,  I would only store `cairoTyp` string representation

Reply:
Would you prefer that I instead wrote `cairoType = CairoType.fromSol(…).toString()`? No problem if so, but I just followed what was done for function parameters in `parameterListWriter`

Reply:
Yes, adding `toString()` at the end would make it more clear (at least for me) the purpose of using the CairoType

Reply:
Done